.github/renovate.json5

```diff
@@ -16,6 +16,6 @@
   // We generally always want the major version
   separateMajorMinor: false,
 
-  // We manually update digest dependencies (eg. hashes in Github actions)
+  // We manually update digest dependencies (eg. hashes in GitHub Actions)
   digest: {enabled: false},
```

Comment:
Renovate supports updating hashes in GitHub Action files. So I recommend letting Renovate update them for you. Doing this by hand is error-prone, and you're likely to forget to check regularly.

I recommend dropping the `digest: {enabled: false},` bit, and instead extending from the `helpers:pinGitHubActionDigests` preset, or the `helpers:pinGitHubActionDigestsToSemver`, depending on your specific pinning needs. :wink: 

Relevant links:

- [Renovate docs, upgrade best practices, pining GitHub Actions](https://docs.renovatebot.com/upgrade-best-practices/#extends-helperspingithubactiondigests).
- [Renovate docs, `helpers:pinGitHubActionDigests` preset](https://docs.renovatebot.com/presets-helpers/#helperspingithubactiondigests)
- [Renovate docs, `helpers:pinGitHubActionDigestsToSemver` preset](https://docs.renovatebot.com/presets-helpers/#helperspingithubactiondigeststosemver)

Reply:
We don't use digests in GH actions, we use always version numbers. Ocasionally we've added a temporal digest to fix a problem and we don't want to update it automatically.